core/tests/webdriverio_utils/LearnerDashboardPage.js

```diff
@@ -114,6 +114,38 @@ var LearnerDashboardPage = function () {
     expect(titleOfExplorationSummary).toMatch(title);
   };
 
+  this.removeExplorationSummaryTileFromPlaylist = async function (title) {
+    let removeTile = await $(`.e2e-test-exp-summary-tile-title=${title}`);
+    await removeTile.moveTo();
+
+    let removeFromPlayLaterListButton = $('.e2e-test-remove-from-playlist-btn');
+    await removeFromPlayLaterListButton.waitForClickable({timeout: 5000});
+    await action.click(
+      'Remove from playlist button',
+      removeFromPlayLaterListButton,
+      false
+    );
+    let confirmDeleteInteractionButton = $(
+      '.e2e-test-confirm-delete-interaction'
+    );
+    await confirmDeleteInteractionButton.waitForClickable({timeout: 5000});
+    await action.click(
+      'Confirm Delete Interaction button',
+      confirmDeleteInteractionButton
+    );
+    await waitFor.invisibilityOf(
+      confirmDeleteInteractionButton,
+      'Delete Interaction modal takes too long to close'
+    );
+  };
+
+  this.expectExplorationTileToBeRemovedFromPlaylist = async function (title) {
+    this.removeExplorationSummaryTileFromPlaylist(title);
```

Comment:
This call should happen in `core/tests/webdriverio_desktop/learnerDashboard.js`, not in the expect function. The `describe` block should look something like this:

```js
    await learnerDashboardPage.removeExplorationSummaryTileFromPlaylist(
      EXPLORATION_FRACTION
    );
    await learnerDashboardPage.expectExplorationTileToBeRemovedFromPlaylist(
      EXPLORATION_FRACTION
    );
    await learnerDashboardPage.removeExplorationSummaryTileFromPlaylist(
      EXPLORATION_SINGING
    );
    await learnerDashboardPage.expectExplorationTileToBeRemovedFromPlaylist(
      EXPLORATION_SINGING
    );
```

Feel free to change up the ordering, but the important point is that calls to `removeExplorationSummaryTileFromPlaylist()` and `expectExplorationTileToBeRemovedFromPlaylist` should both be in the `describe` block. That way, it's easy to understand what the test is doing from reading the `describe` block. Does this make sense?

Reply:
Done!